Sources/WebAuthn/WebAuthnManager.swift

```diff
@@ -51,14 +51,14 @@ public struct WebAuthnManager: Sendable {
     /// - Parameters:
     ///   - user: The user to register.
     ///   - timeout: How long the browser should give the user to choose an authenticator. This value
-    ///     is a *hint* and may be ignored by the browser. Defaults to 60 seconds.
+    ///     is a *hint* and may be ignored by the browser. Defaults to 300000 milliseconds (5 minutes).
     ///   - attestation: The Relying Party's preference regarding attestation. Defaults to `.none`.
     ///   - publicKeyCredentialParameters: A list of public key algorithms the Relying Party chooses to restrict
     ///     support to. Defaults to all supported algorithms.
     /// - Returns: Registration options ready for the browser.
     public func beginRegistration(
         user: PublicKeyCredentialUserEntity,
-        timeout: Duration? = .seconds(3600),
+        timeout: Duration? = .milliseconds(300000),
```

Comment:
Now if only we could just specify `.minutes(5)` 😛

Reply:
Upon further reflection (and [reading](https://forums.swift.org/t/se-0329-clock-instant-date-and-duration/53309/14) the original reason why `.minutes()` is not a thing), I think it would be best to specify this as an expression:
```suggestion
        timeout: Duration? = .seconds(5*60),
```

I'll leave the call to you though, as there's nothing explicitly wrong with how it is now, just potentially clearer to whoever is reading the code and not the documentation.

Reply:
Done

Reply:
Thanks!